app/src/main/java/tech/relaycorp/courier/App.kt

```diff
@@ -11,6 +11,15 @@ class App : Application() {
         DaggerAppComponent.builder().build()
     }
 
+    val mode by lazy {
+        try {
+            classLoader.loadClass("tech.relaycorp.courier.AppTest")
+            Mode.Test
+        } catch (e: Exception) {
```

Comment:
I'd suggest to catch a more specific exception (`ClassNotFoundException` in this case) so we don't inadvertently ignore other errors.

Reply:
Right, will change it.